### DISCUSSION TITLE
future ready talent website unavailable

### DISCUSSION THREAD

**Discussion by fatimazafar12** [github/india#410] [Students]
Am I the only one who is unable to access the future ready talent website right now or is everyone facing this problem? It simply says the site can't be reached. I'm continuing my modules on MS Learn but I don't know if it's syncing my progress or not.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by TheSuranaverse** [comment]
@fatimazafar12 join the community above & connect with me, we will help you with all the queries or doubt
